lib/workers/evm_server.rb

```diff
@@ -58,11 +58,14 @@ def start
       exit
     end
 
-    at_exit {
-      # TODO: should this be called on SOFT ints and SystemExit, not SIGINT?
-      # register a shutdown method to run when server exits
-      MiqServer.stop
-    }
+    # TODO: Forked workers will run this at_exit too.
+    # We need to gate this so it only runs in the server/parent
+    # process.
+    # at_exit {
+    #   # TODO: should this be called on SOFT ints and SystemExit, not SIGINT?
+    #   # register a shutdown method to run when server exits
+    #   MiqServer.stop
+    # }
```

Comment:
Need to find a different way to do this since all forked workers inherit this `at_exit`.